plugins/connection/pct_remote.py

```diff
@@ -0,0 +1,396 @@
+            - For behavior matching paramiko<2.9 set this to V(False)
+        vars:
+            - name: ansible_paramiko_use_rsa_sha2_algorithms
+        ini:
+            - {key: use_rsa_sha2_algorithms, section: paramiko_connection}
+        env:
+            - {name: ANSIBLE_PARAMIKO_USE_RSA_SHA2_ALGORITHMS}
+        default: True
+        type: boolean
+      host_key_auto_add:
+        description: 'Automatically add host keys'
+        env: [{name: ANSIBLE_PARAMIKO_HOST_KEY_AUTO_ADD}]
+        ini:
+          - {key: host_key_auto_add, section: paramiko_connection}
+        type: boolean
+      look_for_keys:
+        default: True
+        description: 'False to disable searching for private key files in ~/.ssh/'
+        env: [{name: ANSIBLE_PARAMIKO_LOOK_FOR_KEYS}]
+        ini:
+        - {key: look_for_keys, section: paramiko_connection}
+        type: boolean
+      proxy_command:
+        default: ''
+        description:
+            - Proxy information for running the connection via a jumphost
+            - Also this plugin will scan 'ssh_args', 'ssh_extra_args' and 'ssh_common_args' from the 'ssh' plugin settings for proxy information if set.
+        type: string
+        env: [{name: ANSIBLE_PARAMIKO_PROXY_COMMAND}]
+        ini:
+          - {key: proxy_command, section: paramiko_connection}
+        vars:
+          - name: ansible_paramiko_proxy_command
+      ssh_args:
+        description: Only used in parsing ProxyCommand for use in this plugin.
+        default: ''
+        type: string
+        ini:
+            - section: 'ssh_connection'
+              key: 'ssh_args'
+        env:
+            - name: ANSIBLE_SSH_ARGS
+        vars:
+            - name: ansible_ssh_args
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
+      ssh_common_args:
+        description: Only used in parsing ProxyCommand for use in this plugin.
+        type: string
+        ini:
+            - section: 'ssh_connection'
+              key: 'ssh_common_args'
+        env:
+            - name: ANSIBLE_SSH_COMMON_ARGS
+        vars:
+            - name: ansible_ssh_common_args
+        cli:
+            - name: ssh_common_args
+        default: ''
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
+      ssh_extra_args:
+        description: Only used in parsing ProxyCommand for use in this plugin.
+        type: string
+        vars:
+            - name: ansible_ssh_extra_args
+        env:
+          - name: ANSIBLE_SSH_EXTRA_ARGS
+        ini:
+          - key: ssh_extra_args
+            section: ssh_connection
+        cli:
+          - name: ssh_extra_args
+        default: ''
+        deprecated:
+            why: In favor of the "proxy_command" option.
+            version: "0.1.0"
+            alternatives: proxy_command
```

Comment:
Probably better to leave options `ssh_args`, `ssh_common_args`, `ssh_extra_args` out - little sense in create a new plugin with deprecated features ;-)

Reply:
I first had them left out, but I needed to put them back in: 74c6e1174c67b4467682a49d92d5d77d38e31bb5
Since I'm inheriting from `ssh_paramiko` `ansible-test` is unhappy if they are not present.